submissions/KostiantynOkhorzin/dom/script.js

```diff
@@ -0,0 +1,174 @@
+    },
+    {
+        title: 'Game of Thrones (season 8)',
+        descr: `The eighth and final season of the fantasy drama television series Game of Thrones, produced by HBO, premiered on April 14, 2019, and concluded on May 19, 2019. Unlike the first six seasons, which consisted of ten episodes each, and the seventh season, which consisted of seven episodes, the eighth season consists of only six episodes.
+
+        The season was filmed from October 2017 to July 2018 and largely consists of original content not found in George R. R. Martin's A Song of Ice and Fire series, while also incorporating material that Martin has revealed to showrunners about the upcoming novels in the series, The Winds of Winter and A Dream of Spring. The season was adapted for television by David Benioff and D. B. Weiss.
+        
+        The season was met with mixed reviews from critics, in contrast to the critical acclaim of previous seasons, and is the lowest-rated of the series on the website Rotten Tomatoes. While the performances, production values and music score were praised, criticism was mainly directed at the shorter runtime of the season as well as numerous creative decisions made by the showrunners regarding the plot and character arcs; many commentators deemed it to be a disappointing conclusion to the series.
+        
+        The season received 32 nominations at the 71st Primetime Emmy Awards, the most for a single season of television in history. It won twelve, including Outstanding Drama Series and Outstanding Supporting Actor in a Drama Series for Peter Dinklage.`,
+        img: 'img/season8.jpg'
+    },
+];
+
+function showSeasonsGot(data) {
+
+    const listBtns = document.querySelector('.side-menu');
+    const mainContent = document.querySelector('.content');
+    const btnBurger = document.querySelector('.burger');
+    const body = document.body;
+
+    const renderBtns = (arr, list) => {
+        for (let i = 0; i < arr.length; i++) {
+            list.insertAdjacentHTML('beforeend', `
+                <li class="side-menu__item">
+                    <button class="side-menu__btn" data-season='${i}'>Season ${i+1}</button>
+                </li>
+            `);
+        }
+    };
+
+    const renderContent = (i = 0, arr = data) => {
+        const {title, descr, img} = arr[i];
+        mainContent.innerHTML = `
+            <h1 class="content__title">${title}</h1>
+            <div class="content__descr">${descr}</div>
+            <div class="content__img-wrapper">
+                <img class="content__img" src=${img} alt=${title}>
+            </div>
+        `;
+    };
+
+    const toggleActiveClass = (i = 0, itemSelector = '.side-menu__btn', activeClassSelector = 'side-menu__btn_active') => {
+        const btns = document.querySelectorAll(itemSelector);
+        btns.forEach(item => {
+            item.classList.remove(activeClassSelector);
+        });
+        btns[i].classList.add(activeClassSelector);
+    };
```

Comment:
I recommend to create a variable `styleList` which have properties with className. Because if in css some class was changed in code we must found and edit all rows with this className.
```
const styleList = {
   activeItemMenu: 'side-menu__btn_active',
    ...........................
}

    const toggleActiveClass = (activeBtn = 0) => {
        const btnList= document.querySelectorAll('.side-menu__btn');
        btnList.forEach(btn=> {
            if(btn.dataset.id !== activeBtn) {
                    btn.classList.remove(activeClassSelector);
            } else {
                     btn.classList.add(activeClassSelector);
            }
        });
    };
```